packages/editor-ui/src/views/__tests__/WorkflowsView.test.ts

```diff
@@ -21,6 +21,8 @@ describe('WorkflowsView', () => {
 	let usersStore: ReturnType<typeof useUsersStore>;
 	let projectsStore: ReturnType<typeof useProjectsStore>;
 
+	const routerReplaceMock = vi.fn();
```

Comment:
Can you move this test file next to the source? (as we discussed last time on the frontend meeting)